scripts/README.md

```diff
@@ -0,0 +1,17 @@
+# Scripts included for the Docker image
+
+The following files are present in this directory for the purposes of the Docker image, as they are common to all instances of the APEL cloud server.
+
+* run_cloud_summariser.sh 
+* apeldbloader-cloud
+
+They could also be included in Server RPMs in the future, so have been included here (rather than the docker directory, which is reserved for files only relevant to the Docker image)
```

Comment:
To clarify: "...in the future, but are not at the moment, so..."

Full stop at end.